book/website/foreword/foreword.md

```diff
@@ -8,8 +8,8 @@ This section has been recently added, and some chapters are still being reviewed
 This is an introductory section of _The Turing Way_ as a project, a handbook, and a community.
 Here you will find chapters providing contexts and background on the book, how you can use it, how you can contribute to different aspects of the project, and how you can participate in this community.
 
-Kirstie Whitaker, founder and the project lead until November 2024, discusses "Emergent Strategy" and how it has inspired her leadership and the foundation of _The Turing Way_.
-Malvika Sharan, current project lead also shares her reflections from working with the community and how _The Turing Way_ applies approaches that underpin the "Principles of Digital Commons."
+Kirstie Whitaker, founder and the project lead until November 2024, discussed "Emergent Strategy" and how it has inspired her leadership and the foundation of _The Turing Way_.
+Malvika Sharan, a previous project lead until Jun2 2025 also shared her reflections from working with the community and how _The Turing Way_ applies approaches that underpin the "Principles of Digital Commons."
```

Comment:
Is this correct to list Malvika as a previous project lead? Also I think that the present tense is correct for discusses and shares....


```suggestion
Malvika Sharan, a previous project lead until June 2025 also shares her reflections from working with the community and how _The Turing Way_ applies approaches that underpin the "Principles of Digital Commons."
```